Set bypassL1 correctly

grypp opened the pull request:
Currently `bypassL1` is always set when using mma.sync. It worked fine because upstream mlir was ignoring it when it cannot be used.

Fix for llvm integrate

nicolasvasilache reviewed (commented):
Can this be tested somehow ?

Or should this fold with @kuhar 's integrate ?

grypp committed:
Set bypassL1 correctly

Currently `bypassL1` is always set when using mma.sync. It worked fine because upstream mlir was ignoring it when it cannot be used.

kuhar reviewed (approved):
Thanks, @grypp!
